cranelift/filetests/filetests/egraph/make-icmp-tests.sh

```diff
@@ -18,6 +18,80 @@ target x86_64
 ;; run with the 'CRANELIFT_TEST_BLESS=1' env var set to update this file
 
 EOF
+
+    # Reflexivity/irreflexivity of `icmp`` cond codes:
+    # for reflexive cond codes,   (icmp cc x x) == true.
+    # for irreflexive cond codes, (icmp cc x x) == false.
+    for cc in $CCS; do
+        cat << EOF >> $out
+function %icmp_${cc}_self(i32) -> i8 {
+block0(v0: i32):
+    v1 = icmp ${cc} v0, v0
+    return v1
+}
+
+EOF
+    done
+
+    # Masking the result of a comparison with 1 always results in the comparison
+    # itself (comparisons in wasm may sometimes be hidden behind extensions):
+    cat << EOF >> $out
+function %mask_icmp_result(i64, i64) -> i8 {
+block0(v1: i64, v2: i64):
+    v3 = icmp ult v1, v2
+    v4 = iconst.i8 1
+    v5 = band v3, v4
+    return v5
+}
+
+function %mask_icmp_extend_result(i64, i64) -> i64 {
+block0(v1: i64, v2: i64):
+    v3 = icmp ult v1, v2
+    v4 = uextend.i64 v3
+    v5 = iconst.i64 1
+    v6 = band v4, v5
+    return v6
+}
+
+EOF
```

Comment:
Let's not put unparameterized tests in script-generated tests. I think these two tests should either stay in `algebraic.clif` or go into a new file.